doc/whats_new/v0.1.rst

```diff
@@ -28,6 +28,7 @@ Changelog
 
 - |Feature| Implement continuous integration and repository docs, by `Adam Li`_ (:pr:`15`)
 - |Feature| Implement conditional independence tests under the ``dodiscover.ci`` submodule, by `Adam Li`_ (:pr:`16`)
+- |Feature| Implement PC algorithm and skeleton for constraint-based structure learning algorithms under the ``dodiscover.constraint`` submodule, by `Adam Li`_ (:pr:`18`)
```

Comment:
@adam2392 are you updating this change log manually?  Isn't this something we could handle with semantic commits?  @darthtrevino

Reply:
Yeah rn it's just a per PR changelog. What's semantic commits? :0

Reply:
https://python-semantic-release.readthedocs.io/ - this library will parse git commit messages for changelog information

https://pypi.org/project/semversioner/ - this project will let you batch up change documents that are applied into the changelog via their CLI.

I kind of prefer the latter approach; commit-message parsing can be flaky if contributors aren't aware of the requirement. We use similar tooling in our JS projects, and we require that every PR has a semver-impact document associated with it.

Reply:
> commit-message parsing can be flaky

I've certainly been guilty of crummy semantic commit messages.  I'm inclined to go with whatever you think is best @darthtrevino .  I'd like to employ what CI tech that can help these PR's can get leaner.  The leaner they are, the easier it is for me to onboard new contributors.